api/nodes/serializers.py

```diff
@@ -373,3 +379,29 @@ class Meta:
     @staticmethod
     def get_id(obj):
         return '{}:{}'.format(obj.node._id, obj.provider)
+
+class NodeInstitutionRelationshipSerializer(JSONAPIRelationshipSerializer):
```

Comment:
With the changes on the parser, then `id` and `type` can be fields on the serializer.  `id` would be a CharField and `type` a TypeField.  The fields would have required=False, and maybe allow_null=True?

The TypeField would take care of ensuring the institution is the correct type, and throw a 409 error.

These changes on the parser and this serializer, to allow id and type to be fields on the serializer are really matters of opinion, but I think help keep things consistent with how our current parser works.